newsroom/agenda/agenda.py

```diff
@@ -334,42 +353,12 @@ def _set_event_date_range(search):
     should = []
 
     if date_from and not date_to:
-        dates_field = "dates.start" if app.config.get("AGENDA_SHOW_MULTIDAY_ON_START_ONLY") else "dates.end"
         # Filter from a particular date onwards
-        should = [
-            {
-                "bool": {
-                    "must_not": {"term": {"dates.all_day": True}},
-                    "filter": {"range": {dates_field: {"gte": date_from}}},
-                },
-            },
-            {
-                "bool": {
-                    "filter": [
-                        {"term": {"dates.all_day": True}},
-                        {"range": {dates_field: {"gte": search.args["date_from"]}}},
-                    ],
-                },
-            },
-        ]
+        dates_field = "dates.start" if app.config.get("AGENDA_SHOW_MULTIDAY_ON_START_ONLY") else "dates.end"
```

Comment:
I think we wouldn't need this config anymore if we show those hidden items. that's like doing both at the same time.